crates/services/consensus_module/poa/src/service.rs

```diff
@@ -609,7 +782,21 @@ where
 }
 
 #[allow(clippy::too_many_arguments)]
-pub fn new_service<T, B, I, P, S, PB, C>(
+pub fn new_service<
+    T,
+    B,
+    I,
+    P,
+    S,
+    PB,
+    C,
+    TxRcv,
```

Comment:
Maybe prefix them all with a slug to mean it's generics for preconfirmation like `PC`